src/integration/java/uk/nhs/prm/repo/ehrtransferservice/repo_incoming/EhrRequestTest.java

```diff
@@ -74,15 +89,68 @@ void Given_ValidRepoIncomingEvent_When_PublishedToRepoIncomingQueue_Then_CreateC
                 .willReturn(aResponse().withStatus(204))
         );
 
-        queueUtility.sendSqsMessage(repoIncomingMessage, repoIncomingQueueName);
+        SQSQueueUtility.sendSqsMessage(repoIncomingMessage, repoIncomingQueueName);
 
         // then
-        await().atMost(20, TimeUnit.SECONDS).untilAsserted(() -> {
-            final ConversationRecord record = transferService
-                .getConversationByInboundConversationId(INBOUND_CONVERSATION_ID);
+        waitForConversationTransferStatusMatching(INBOUND_REQUEST_SENT);
+    }
 
-            assertEquals(INBOUND_REQUEST_SENT.name(), record.state());
-        });
+    @Test
+    void Given_ValidRepoIncomingEventForSmallEhr_When_NoEhrResponseReceived_Then_UpdateStatusToInboundTimeout() {
+        // given
+        // override inboundTimeoutSeconds so that the request will timeout within the timeframe
+        System.setProperty("inboundTimeoutSeconds", "10");
+
+        final String repoIncomingMessage = getRepoIncomingMessage();
+
+        // when
+        stubFor(post(
+                urlMatching("/health-record-requests/" + NHS_NUMBER))
+                .withHeader("Authorization", equalTo(gp2gpMessengerAuthKey))
+                .withHeader("Content-Type", equalTo("application/json"))
+                .willReturn(aResponse().withStatus(204))
+        );
+
+        // send request
```

Comment:
I don't think this comment is necessary, the code is readable and easy to understand!